source/release-notes/1.1.txt

```diff
@@ -0,0 +1,48 @@
+.. _c2c-release-notes-1.1:
+
+===================================
+Release Notes for ``mongosync`` 1.1
+===================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   ``mongosync`` 1.1 Released January 13, 2023
+
+
+Filtered Sync
+-------------
+
+.. include:: /includes/api/facts/filtering-intro.rst
+
+For more information, see: :ref:`c2c-filtered-sync`.
+
+Unlike Sharded Topologies
+-------------------------
+
+{+c2c-product-name+} 1.1 supports sync between clusters with different
+sharding configurations.
+
+You can:
+
+- Change the number of shards from M to N
+- Sync between two sharded clusters
+- Sync from a replica set to a sharded cluster
```

Comment:
Can we change this to:

```
- Sync between two sharded clusters with different numbers of shards
- Sync from a replica set to a sharded cluster
```

I don't like "Change the number of shards from M to N" since that could be misunderstood as being able to change the shard numbers during sync.

Reply:
Done